CHANGELOG.rst

```diff
@@ -5,6 +5,7 @@ Version History
 v5.14.11
```

Comment:
I have one question, should this be a minor change? I think on the change of daphne per uvicorn... should that affect on the numeration?

Reply:
If the outward appearance of the application has not changed, then a minor version is fine. Internals are allowed to changed within that context.

Reply:
this is a simple and great explanation! gotcha :+1:.

Reply:
The minor version is the second number in the semver. The last number is the patch version. The server replacement warrants a minor not a patch level change.

Reply:
thanks for catching this, it almost missed that.